src/unittest/test_moveaction.cpp

```diff
@@ -47,16 +46,36 @@ class TestMoveAction : public TestBase
 
 static TestMoveAction g_test_instance;
 
+const static char *helper_lua_src = R"(
```

Comment:
This is not very editing-friendly (lack of syntax highlight) and it'll also get noticeably longer in #12595. Thus it might be better to make this test optional in case the file was not found.

Reply:
Then you still have a hardcoded path...

Another option would be to move it to `builtin`. It would always be available then.

Reply:
Hmm yes. Either builtin or abusing devtest. I will change it accordingly in my PR.